oitg/fitting/decaying_sinusoid.py

```diff
@@ -47,8 +47,11 @@ def init_all(t, y, p_dict):
     # and smaller than the data time range. (Niquist and Fourier limit)
     min_step = np.min(t[1:] - t[:-1])
     duration = t[-1] - t[0]
+    # Niquist limit does not apply to irregularly spaced data
+    # We'll use it as a starting point anyway...
     f_max = 0.5 / min_step
-    f_min = 0.5 / duration
+    # relaxed Fourier limit
```

Comment:
Just as an aside (I realize that this was introduced earlier already), I think "Fourier limit" is rather non-standard terminology. I have only ever come across this term in a completely different context (transform-limited pulses). 

Also, while it is true that the resolution of the Fourier transform is limited by the (inverse) acquisition time, this isn't really a limit for fits in the same way that the Nyquist theorem is – it limits the lowest frequency in Fourier analysis because of the implicit assumption of a periodic input signal. If the signal is known to be described by a sine wave, an arbitrarily low frequency can be fitted even for short acquisition time windows (with SNR implications, of course).

Still useful as an initial guess, though, as users would usually choose the scan range on the input data to encompass a non-negligible fraction of the π time.